### PR TITLE
Sara/mkt 885 list packages in semgrep docker images

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR

### CODE REVIEW

**Comment by s-santillan** [comment]
Render:
image

File: docs/semgrep-ci/packages-in-semgrep-docker.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: Packages included in the latest Semgrep docker image.
+tags:
+  - Semgrep in CI
+---
+
+# Packages in the Semgrep docker image 
+
+## Packages
+
+The [<i class="fas fa-external-link fa-xs"></i> `semgrep/semgrep:latest` docker image](https://hub.docker.com/r/semgrep/semgrep/tags) contains the following packages:
```
Comment (aryx) [inline docs/semgrep-ci/packages-in-semgrep-docker.md:14]: maybe add "contains obviously the 'semgrep' program but also ..."

File: docs/semgrep-ci/packages-in-semgrep-docker.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- `bash`
+- `jq`
+- `curl`
+- Python 3.11 (`3.11-alpine` base image)
+
+The Python 3.11 docker includes additional packages (for example, `ncurses`) that can change without notice. To review them, see the following links:
```
Comment (aryx) [inline docs/semgrep-ci/packages-in-semgrep-docker.md:21]: maybe add after "that can change without Notice. IMPORTANT: you should not rely on their presence in your CI workflow. Only the packages listed above are officially supported and guaranteed to be available in further version of the docker images"

**Review by aryx — APPROVED** [review]

**Comment by werrett** [comment]
Do we want to call out `python 3.11`? A quick search says we should be updating it once a year (and presumably updating these docs too).,

**Review by werrett — COMMENTED** [review]
LGTM

**Pull request merged**